src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -5830,8 +5830,13 @@ class A                                                               \
             Assert.False(result.ContainsErrors, $"Compilation error(s) occurred: {result.Output} {result.Errors}");
 
             var assemblyName = AssemblyName.GetAssemblyName(Path.Combine(tempDir, name));
+#if NETCOREAPP
```

Comment:
Instead of testing that `AssemblyName.ToString()` generates a string a certain way, could we change this test to assert the AssemblyName properties are as expected?